src/libvirtApi/snapshot.js

```diff
@@ -31,7 +31,11 @@ import { logDebug } from '../helpers.js';
 
 export function snapshotCreate({ connectionName, vmId, name, description, memoryPath, disks, isExternal, storagePools }) {
     // that flag ought to be implicit for non-running VMs, see https://issues.redhat.com/browse/RHEL-22797
-    const flags = (!isExternal || memoryPath) ? 0 : Enum.VIR_DOMAIN_SNAPSHOT_CREATE_DISK_ONLY;
+    let flags = 0;
+    if (isExternal && !memoryPath)
+        flags |= Enum.VIR_DOMAIN_SNAPSHOT_CREATE_DISK_ONLY | Enum.VIR_DOMAIN_SNAPSHOT_CREATE_QUIESCE;
+    if (memoryPath)
+        flags |= Enum.VIR_DOMAIN_SNAPSHOT_CREATE_LIVE;
```

Comment:
I know this is a draft, and maybe it's just a reminder. But this can't go in like this. We should always support the non-live way where a running VM is paused first before taking the memory dump. That is slower (and some connections to the VM may time out during the pause), but much safer and predictable. The live option should be behind some big :warning:  "danger! don't do this unless you are know what you are doing!" kind of warning sign. That's the one which has indefinite runtime and disk requirements and thus is hard to predict, no?